indico/modules/rb_new/client/js/setup.jsx

```diff
@@ -11,7 +11,7 @@ import {selectors as configSelectors} from './common/config';
 import {selectors as userSelectors, actions as userActions} from './common/user';
 import {actions as roomsActions} from './common/rooms';
 
-import 'semantic-ui-css/semantic.css';
+import 'indico-sui-theme/dist/semantic.css';
```

Comment:
any easy way to skip the `dist/` when building the theme package?

Reply:
We could put the dist files in the package root. But is it that bad that you get to see the `dist/`?

Reply:
just seems a bit ugly to have this in the import since it's the only thing from the package that ever gets imported :see_no_evil: